mne_connectivity/viz/tests/test_3d.py

```diff
@@ -27,7 +30,16 @@ def test_plot_sensors_connectivity(renderer):
         plot_sensors_connectivity(info='foo', con=con, picks=picks)
     with pytest.raises(ValueError, match='does not correspond to the size'):
         plot_sensors_connectivity(info=info, con=con[::2, ::2], picks=picks)
-
-    fig = plot_sensors_connectivity(info=info, con=con, picks=picks)
+    cmap = 'viridis'
+    fig = plot_sensors_connectivity(info=info, con=con, picks=picks, cmap=cmap)
+    # check colormap
+    cmap_from_mpl = np.array(colormaps[cmap].colors)
+    cmap_from_vtk = np.array(
+        fig.plotter.scalar_bar.GetLookupTable().GetTable()
+    )
+    cmap_from_vtk = cmap_from_vtk[:, :3] / cmap_from_vtk[:, [-1]]
```

Comment:
This was actually a dumb way to do it (relies on the fact that alpha channel happens to be 255 for all steps of the color ramp). Better:

```suggestion
    # discard alpha channel and convert uint8 -> norm
    cmap_from_vtk = cmap_from_vtk[:, :3] / 255
```